op-proposer/flags/flags.go

```diff
@@ -60,6 +60,12 @@ var (
 		Usage:   "Interval between submitting L2 output proposals when the dispute game factory address is set",
 		EnvVars: prefixEnvVars("PROPOSAL_INTERVAL"),
 	}
+	ProposalRetryIntervalFlag = &cli.DurationFlag{
+		Name:    "proposal-retry-interval",
+		Usage:   "Interval between retrying output proposals if one fails",
+		Value:   120 * time.Second,
```

Comment:
```suggestion
		Value:   time.Minute,
```
I think 1 minute is also OK.